src/index.html

```diff
@@ -4,13 +4,32 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+    class="search-form"
+    data-qa="big">
+      <div class="search-icon"></div>
+      <input
+      type="text"
+      class="search-form__input search-form__input--big"
+      placeholder="Try “Los Angeles”"
+      data-qa="keypress">
+    </form>
```

Comment:
format next one in same way
```suggestion
    <form class="search-form" data-qa="big">
      <div class="search-icon"></div>
      <input
        type="text"
        class="search-form__input search-form__input--big"
        placeholder="Try “Los Angeles”"
        data-qa="keypress"
      >
    </form>
```